src/assets/social-media-icons/linkedin.svg

```diff
@@ -0,0 +1,11 @@
NxR4AMJRa63lWP6bsh00+hmyX7gm+72uts/v+rNgDAIbRrtP7acWnftyXrdu7tPl/WfGpd+0egit5Ni4AMJKLFa/99thbq7SnVjz6nnYrXCb5kr+eqfusZ92WUs5a0N5euXyRxd/7ZOWf8WxcAGALdv5s3Lad+evSy5+zuI/eoyKr1jrP6q3g5/otyYfnrDK2VbzLfH345F+rnvxhGxcA2HvtVOqq1bvZc1fTNuxtkl/b48+etHLY7gu46rYxK1cvxR4AMIKzfL3S9duET96+zhOehnGjrQx+XHr51arDGmIPABjBqpWuyd40uXn00zCWrPr7nS+/IPYAgL3WVrOWV/Xeb+MxaOURknybxe1SVj327MaLrHGvvDtmuVzxvV/VWk9vvyD2AIB9t9YK166VUq5KKRftsWffJvl0x5e+ecbq3srrFG//RuwBAHurnUx9vfTyVlb1nqPNc5LF6eBVZk/8vpf5+tq9t7cfoyb2AIB9Nlvx2sWOZ1hLOxV8fsenn3NPv4sVr/25lSv2AIB9drr0+88TPoGbLJ7Zu8qT7+nXTuZeL70s9gCA/da2Kpe3cO+KqUnY4j3/5ku/f3vzH8QeALCvTla8Nt/xDFPxVeS2p6SJPQBgb311nduqx4UdiPmK144TsQcA7K+Tpd/fdWuTybjnFivL19w9Sjvtu/w9xB4AsNeOln5/2WOIR7rrqR7zDXzv5b//USL2AID99Wrp91c9hlhXe9LH93d8ehPbz/Ol379JxB4AsIfazZSXTXJlr9b6stZ6nuTXO77kOls8RfxNJvqDAQD22tWWv//Rite2dVuTP92ccF3DUfs4ycP30Dvf0C1ZVjbdN6WUL7XWDXx/AICFqT2ubIN+3/D3e19KWfV826f4KhhrrSc327iTP70CAOyNu57/yt+9L6XMtv0mN7F3te03AgAOxlXvASbuOskPuwi9ZHHNXrLY43173xcCAKxp3nuACfqcRW99SPJhi49NW/blJvbmSX7a0ZsCAGMb+fDnz3e8/jLJv1e8/lsp5XSL89x2tPxCKeXy9soeAMAmzHfwHqva5Wjbb1pKOb/rc+12MMs7pW9rrac7eozb0aoX/5EkbSnx4w6GAADG9mkXW5R3vMfRtt/3AbOsfuzZRa315Q7ef/k9Pid/v6nyoT44GADYnIsdvtfy3UTueu7sTrQAXfU4tBfZzc9l+e9/lYg9AGCzdtkTV0u/7xp7SVJKucjq3dK3tdZtX7u3fPPmy+RW7LWbH9rKBQCe6tOOb6a8fN3eqzseo7Zrs+x4O7fWuip0/x57N0NsYwAA4CBs6kkQ65qveO1kxzN8pQXvqp/FNrdzT1a8dpkkZfnVWuuXNgwAwLquSym7OITwN/XrZ75u5KkUtdZ5VjzTtpTyVTvd8z0uk7xe8al/bfp07or3+vOfx/LKXrL7KgcA9l+vfvht6fenOzr5uo7ZHa9vdDu3bV0vR+WfMXlX7K3aZwYAWOU6/WJveYXsRZJd3cT4XqWUyyS/rPjUprdzV/197469dmzY6h4AsK53O3z817JV26GzXQ9xj/O0+90t2eTp3OXbvVzf3iZetbJ3c3foVYMBANz2+b6nSmxbi8zlrdw3tdaTDuN8pc03u+PTz97OrbXOkrxaevlvAbwy9prZc94cADgIs94DZPWO5Pmuh7hLKWWer4M02cx27vlDr90Ze22w988cAAAY1/vWC121GZafpjGZ1b1mltVnIt4+dc47VvU+Lt/r8L6VvWSxB2w7FwBY9jmrHw3Wy6rVvcmcQbjnUWrJE7Zz29efr/jUV6/dG3ttsEmcaAEAJuW046GMr9zxmLLXtdbz3U+z2j2PUnuVx287n2f1qt58+QsfWtm7OTb8wyMHAADG9UPrg6k5X/HaT3c8SqyXWVZv5/573e3c9nX/vuN7f+Wbdb5pKeWi/aBWfWMA4HD80laoJqeUMq+1vk/y/dKnLmqtJ49cibzI6sexPUsp5apda7cqQI8e+vNt+3bV7WZ+vuu5xGs/8qO9wUW+/gECAIdhI48i26YWQ1f5+tGvv5VS9v7StDsewfaplHLn6uWD27i3tX/ATugCwOGZfOgl9543eNsWrfZWm3/Vs3Zn9/25R8VeIvgA4ADtRejdaIcUfl7xqe/3Nfju2V198PrJR23jLr3pWZL/PPXPAwB74cdSymRuYfIYtdYPSd6u+NRexes9obfW3+PJsdfe/DSLCxiX98UBgP12nWR2+xmr+6ZdvzfP6q3P90nOpnT7mGW3DmO8WfHpta9BfFbstUGOsgi+VYMAAPvnYxahd9V7kOd6IPg+ZXG/wKtdzrSOdheUi9w999qnix99zd6yUspVKeUki73xVfeNAQD2w3UW27YnUwygp2hBdJKvH6eWLELqst0KZTLapXLzbCD0kg2s7N3WVvneZfX+OAAwXe+TnI8Secse2BJNJrCa2Vbz3uXuGZ90reGzV/Zua6t8p4F7sIQAAAFqSURBVEm+y+rHgQAA0/IxyXellCG2be9SSvnSdiJ/ueNL3iT5o9Z60RavdqbWetQOYfwvd4fez089VLLRlb1l7XEes7gRMwBMzfskF6uepTq6NQ+Yvk/ybpuPhVuzk66zuK5w/tT32Wrs3WhLp7P2sWr/GQDYvk9ZRM7FlE+h7sIjDpje/Mw+bGLls23VniQ5S/LqgS//LYut5Wf9s9pJ7N3WfrgnWdzd+jgP/0UBgKf5nMWF/vMk85G3aZ+qHc44z3o9cvPzvLz5uC/EWvMcZdE7N5G37vvMNrXquvPYW9ZW/W5+CC9v/QoArO+qfXzJGiHCX1qLnLWPp947+FMWP/ujPH0h63MWh2QunvjnAQC4S631Za31rNZ6VXdrcrd/AQAYWq31pC5O5n7ZUuBd1Vrf1cU1fFvVfRsXAGDKWpCdZnHN3XGettV7nb9fP7m1U77LxB4AwCPUv84bHLWPu1ymXUPZ8/rJ/w91NpKdLApiLAAAAABJRU5ErkJggg=="/>
```

Comment:
Change it as real SVG path or convert it to png.

\cc @delisma

Reply:
I took care of avoiding the most <g> tag nesting as possible, but there is still a need for <g> tags containing a transform attribute to be nested. I also cleaned up the unnecessary id attributes.

In my recent force push, I did not convert SVGs into PNGs or replaced PNGs by SVG equivalent lines.

Reply:
I kept everything SVG only as SVG, and converted LinkedIn back to a PNG since the logo was hidden as a PNG inside the SVG